conda/recipes/legate-dataframe/meta.yaml

```diff
@@ -89,7 +89,7 @@ test:
   # import tests at build time do not work for the CUDA packages,
   # because builds happen on machines without a GPU.  So no tests.
   commands:
-    - pip show legate-dataframe
+    - pip show legate-dataframe-cu12
```

Comment:
This `-cu{11,12}` convention shouldn't be used for conda packages, since conda has a better solution for differentiating between CUDA versions (the `__cuda` virtual package).

To prevent these suffixes from making it into conda packages, across RAPIDS we pass this to `pip install` in conda builds:

```text
--config-settings rapidsai.disable-cuda=true
```

Example from `cudf`: https://github.com/rapidsai/cudf/blob/1b82963df736f3ad71b003443a4de1414f3ce2e5/build.sh#L78

I recommend doing the same here.